### ISSUE TITLE
Read Speaker on Android is always reading out back button as "nagivate up"

### ISSUE THREAD

**Issue by lfaz** [wix/react-native-navigation#7221]
## 🐛 Bug Report

`accessibilityLabel` for backButton on Android is always reading as "navigate up" whatever you add in the options object for ex:

```
const options = {
  topBar: {
    backButton: {
      accessibilityLabel: "Custom message"
    },
  },
};
```

The above does not seem to work for me as stated in `https://wix.github.io/react-native-navigation/api/options-backButton/#accessibilitylabel`. Am I missing something, or how does one add accessibilityLabel for back button? I also tried to add into backButton object inside packages/navigator-config/styles.js but still android speaker reads it as **"navigate up"**.

Removing line 37 from file node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/options/BackButton.java seems to fix this but ofc this is just some work around and not aware of possible side effects! 

Also what about iOS part? Is there any way to write a custom message for back button accessibility?

## To Reproduce

1. Run library on android device...
2. Turn on the talk back on android
3. Press navigate back button 

## Expected behavior

Should read as set in `accessiblityLabel` property

(Write what you thought would happen.)

## Actual Behavior

Read speaker reads as navigate up always

## Your Environment

- React Native Navigation version: 7.18.0
- React Native version: 0.64
- Platform(s) (iOS, Android, or both?): Android
- Device info (Simulator/Device? OS version? Debug/Release?): Pixel 4 Emulator

### Are you willing to resolve this issue by submitting a Pull Request?

 - ✖️  &nbsp; Yes, I have the time, but I don't know how to start. I would need guidance.

**Comment by swabbass** [comment]
Hey thanks for reporting, looks interesting,  if you pass `accessibilityLabel` is should be parsed and set as stated [here](https://github.com/wix/react-native-navigation/blob/master/lib/android/app/src/main/java/com/reactnativenavigation/options/BackButton.java#L22),
It seems that it is related to some flow where the button field got cleared, can you, kindly, try to reproduce your issue in our playground? that will make things more clear.

**Comment by Magnus-V** [comment]
I'm have encountered the same bug, when setting the accessibilityLabel as an OptionsTopBarBackButton it is never registered and reverts to "Navigate up".

This has been on both 7.13.0 and 7.18.1 (tried updating, but problem persists). 
React-native version: 0.63.4.

**Comment by lfaz** [comment]
I already tried this on your playground from master branch and I can say that I am still getting same behaviour.

**Comment by retyui** [comment]
Fixed in: https://github.com/wix/react-native-navigation/pull/7282

**Comment by retyui** [comment]
- Released [7.22.0](https://github.com/wix/react-native-navigation/releases/tag/7.22.0)
- Diff https://npmfs.com/compare/react-native-navigation/7.21.0/7.22.0/
